FutureFeatures.md

```diff
@@ -336,3 +336,26 @@ appeared. Another option would be to offer another non-standard execution mode,
 enabled only from developer tools, that would enable traps on selected floating
 point exceptions, however care should be taken, since not all floating point
 exceptions indicate bugs.
+
+## Integer Overflow Detection
+
+There are two different use cases here, one where the application is prepared
+to handle overflow, and one where it isn't.
+
+When the application is prepared to handle overflow, it would be useful to have
+arithmetic operations which can indicate when overflow occured. An example of
+this is the checked arithmetic builtins available in some compilers:
+
+ * http://clang.llvm.org/docs/LanguageExtensions.html#checked-arithmetic-builtins
+ * https://gcc.gnu.org/onlinedocs/gcc/Integer-Overflow-Builtins.html
```

Comment:
Link to [clang](http://clang.llvm.org/docs/LanguageExtensions.html#checked-arithmetic-builtins) and [GCC](https://gcc.gnu.org/onlinedocs/gcc/Integer-Overflow-Builtins.html) instead of long URLs.